trippo/src/components/navBar/Navbar.tsx

```diff
@@ -11,9 +11,12 @@ import IconButton from "@material-ui/core/IconButton";
 import MenuIcon from "@material-ui/icons/Menu";
 import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
 import ChevronRightIcon from "@material-ui/icons/ChevronRight";
+import AccountCircle from "@material-ui/icons/AccountCircle";
 import ListItem from "@material-ui/core/ListItem";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
 import { withRouter } from "react-router-dom";
-import { BLACK, WHITE } from "../../colors/colors";
+import { BLACK, GREY, WHITE } from "../../colors/colors";
```

Comment:
```suggestion
import { BLACK, WHITE } from "../../colors/colors";
```
if you decide to take out the grey icon

Reply:
yeah I tried with black before and again just now, it just looks too dark to me. I just color picked with console and the other icons are (#656464) and (#89898b). So I think keeping it GREY (#656464) is more consistent?